Cargo.toml

```diff
@@ -25,7 +25,7 @@ ansi_term = { version = "0.12.1", optional = true }
 anyhow = "1"
 async-imap = { git = "https://github.com/async-email/async-imap", branch = "master", default-features = false, features = ["runtime-tokio"] }
 async-native-tls = { version = "0.4", default-features = false, features = ["runtime-tokio"] }
-async-smtp = { version = "0.5", default-features = false, features = ["smtp-transport", "socks5", "runtime-tokio"] }
+async-smtp = { git = "https://github.com/async-email/async-smtp", branch = "master", default-features = false, features = ["smtp-transport", "socks5", "runtime-tokio"] }
```

Comment:
not a merge-blocking comment but is there anything speaking against doing a 0.6 release for async-smtp?